src/components/search/Search.module.scss

```diff
@@ -0,0 +1,35 @@
+.artists-wrapper {
+  padding: 0 10px;
+  text-align: center;
+}
+
+.search-input {
+  width: 100%;
+  min-width: 300px;
+  max-width: 640px;
+  margin-left: 0;
+  margin-right: 0;
+  padding: 9px 10px 8px 32px;
```

Comment:
` padding: 9px 10px 8px 32px;`
values must be a multiple of 5 - is good code style

Reply:
fix: 842d06a0778acdda4c606d16097248d12defd2b2